global.json

```diff
@@ -1,4 +1,9 @@
 {
+  "sdk": {
```

Comment:
Intentional?

Reply:
It was part of some of my attempts (visible in one of the other branches) to get to "passing" builds.  Removed.